### PR TITLE
Modify NAS n_worker as a variable

### PR DESCRIPTION
**Pull request by XuehaoSun**
Signed-off-by: Sun, Xuehao <xuehao.sun@intel.com>

## Type of Change

Modify NAS n_worker as a variable

## Description

Change NAS n_worker to be a variable instead of a constant

## Expected Behavior & Potential Risk

NAS working normally

## How has this PR been tested?

test pass

## Dependency Change?

no

### CODE REVIEW

File: neural_compressor/experimental/nas/dynas.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def __init__(self, conf_fname_or_obj):
         self.macs_predictor = None
         self.latency_predictor = None
         self.results_csv_path = None
+        self.num_workers = 20
```
Comment (macsz) [inline neural_compressor/experimental/nas/dynas.py:75]: Perhaps we should expose `num_workers` to be configurable via config and assign it here - https://github.com/intel/neural-compressor/blob/7c48cac0ba944098df9a55b0799cd6c41bf3866c/neural_compressor/experimental/nas/basic_nas.py#L70 ?

Comment (macsz) [reply neural_compressor/experimental/nas/dynas.py]: Pushed new commit, this should help.

**Comment by chensuyue** [comment]
@ftian1 please review.

**Comment by chensuyue** [comment]
@ftian1 @XinyuYe-Intel please review.

**Review by XinyuYe-Intel — APPROVED** [review]

**Review by macsz — APPROVED** [review]

**Pull request merged**